src/components/AdoptionChart.tsx

```diff
@@ -1,62 +1,54 @@
-// Libraries
-import React, { ReactNode } from "react"
-import { Box, BoxProps, Flex, useColorMode } from "@chakra-ui/react"
+import { useTranslation } from "next-i18next"
+import { Box, type BoxProps, Flex, useColorMode } from "@chakra-ui/react"
 
-// Components
-import Translation from "./Translation"
+import type { ChildOnlyProp } from "@/lib/types"
 
-const Column: React.FC<{ children: ReactNode }> = ({ children }) => {
-  return (
-    <Flex
-      flexDirection="column-reverse"
-      ms={{ base: 2, md: 4 }}
-      _first={{ ms: 0 }}
-    >
-      {children}
-    </Flex>
-  )
-}
+const Column = ({ children }: ChildOnlyProp) => (
+  <Flex
+    flexDirection="column-reverse"
+    ms={{ base: 2, md: 4 }}
+    _first={{ ms: 0 }}
+  >
+    {children}
+  </Flex>
+)
 
-const Cell: React.FC<BoxProps> = ({ children, color, ...props }) => {
-  return (
-    <Box
-      border="1px solid"
-      borderColor={color || "text"}
-      color={color || "text"}
-      py="0.8rem"
-      px={{ base: 2, md: "1.2rem" }}
-      fontSize="0.9rem"
-      fontWeight="bold"
-      lineHeight="none"
-      textAlign="center"
-      _last={{
-        borderTopStartRadius: "2xl",
-        borderTopEndRadius: "2xl",
-      }}
-      sx={{
-        "&:nth-child(-n + 2)": {
-          borderBottomStartRadius: "2xl",
-          borderBottomEndRadius: "2xl",
-        },
-      }}
-      {...props}
-    >
-      {children}
-    </Box>
-  )
-}
-
-const ColumnName: React.FC<{ children: ReactNode }> = ({ children }) => {
-  return (
-    <Cell border="none" pt={6}>
-      {children}
-    </Cell>
-  )
-}
+const Cell = ({ children, color, ...props }: BoxProps) => (
+  <Box
+    border="1px solid"
+    borderColor={color || "text"}
+    color={color || "text"}
+    py="0.8rem"
+    px={{ base: 2, md: "1.2rem" }}
+    fontSize="0.9rem"
+    fontWeight="bold"
+    lineHeight="none"
+    textAlign="center"
+    _last={{
+      borderTopStartRadius: "2xl",
+      borderTopEndRadius: "2xl",
+    }}
+    sx={{
+      "&:nth-child(-n + 2)": {
+        borderBottomStartRadius: "2xl",
+        borderBottomEndRadius: "2xl",
+      },
+    }}
+    {...props}
+  >
+    {children}
+  </Box>
+)
 
-interface IProps {}
+const ColumnName = ({ children }: ChildOnlyProp) => (
+  <Cell border="none" pt={6}>
+    {children}
+  </Cell>
+)
 
-const AdoptionChart: React.FC<IProps> = () => {
+const AdoptionChart = () => {
+  const { t } = useTranslation("page-what-is-ethereum")
+  // TODO: Use useColorModeValue
```

Comment:
Well no.. We use `useColorMode` to fetch the color mode, then assign to an `isDark` boolean, then use that for the logic below to differentiate.... this TODO was to switch this to a cleaner `useColorModeValue(lightValue, darkValue)`. Can just remove this in as a code comment though and leave as a potential side task we can chat about, since this may-or-may-not be desired here.

Reply:
Ahh got it. I thought that we were using `useColorModeValue` already. Make sense.